examples/mini_librispeech/mini_librispeech.ipynb

```diff
@@ -68,6 +69,26 @@
     "mini_librispeech_manifests = prepare_mini_librispeech(corpus_dir, output_dir)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# [Optional] Data augmentation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "use_data_augmentation = False\n",
+    "augmenter = WavAugmenter.create_predefined('pitch_reverb_tdrop', sampling_rate=16000) if use_data_augmentation else None\n",
+    "# It seems when spawning multiple Python subprocesses with the same sox handle it raises \"std::runtime_error: Couldn't close file\"\n",
+    "# For now, work around \n",
```

Comment:
perhaps could raise a bug with the sox package maintainer?

Reply:
Actually, it seems to be specific to my jupyter lab setup; it works well in the command line, and in Google colab using a modified official WavAugment example notebook... If I come up with a way to make this reproducible, I'll raise it with them.